blivet/populator/populator.py

```diff
@@ -390,6 +390,17 @@ def teardown_disk_images(self):
             loop_device = dm_device.parents[0]
             loop_device.teardown()
 
+    def _teardown_partially_hidden_vgs(self):
+        """ Teardown VGs that are not completely hidden (one of the PVs
+            is not hidden)
+        """
+
+        for device in self._hidden:
+            if device.type == "lvmvg":
+                if not all(pv in self._hidden for pv in device.pvs):
+                    log.info("Deactivating partially hidden VG '%s'", device.name)
+                    device._teardown()
```

Comment:
Why `_teardown()` and not `teardown()`?

Reply:
Because we can't run `StorageDevice._pre_teardown()` -- it checks VG `status` and we have "missing" PVs so it's False and because of this `_pre_teardown()` will return before actually running `_teardown()`.

Reply:
That's worth a comment in the code, I believe.

Reply:
Ok, comment added.